app/views/outfits/show.html.erb

```diff
@@ -1,21 +1,98 @@
-<h2><%= @outfit.title %></h2>
-<h4>for<%=  @outfit.animal %></h4>
-
-<div>
-    <% if @outfit.occasion != nil %>
-      <p>Special Occasion: <%=@outfit.occasion %></p>
-    <% end %>
-    <p>Size: <%= @outfit.size  %></p>
-    <p>Description: </p><%= @outfit.description %></p>
-</div>
-
-<div>
-  <% @outfit.photos.each do |photo| %>
-    <%= cl_image_tag photo.key, height: 400, width: 400, crop: :fill %>
-  <% end %>
-</div>
-
-<div>
-  <h4>Price per day: $<%= @outfit.price_per_day %></h4>
-</div>
-<%= link_to "Book now ! ",  new_outfit_booking_path(@outfit) %>
+<div class="container-pet mt-5 mb-5">
+        <div class="row d-flex justify-content-center">
+            <div class="col-md-8">
+                <div class="card-outfit">
+                        <div class="text-left paw p-2 px-5">
+                            <img src="https://upload.wikimedia.org/wikipedia/commons/5/51/Paw-print.svg" width="50">
+                        </div>
+                        <div class="invoice p-5">
+                            <h2>Your next Pawsh outfit!</h2>
+                            <h4> for a <%=  @outfit.animal %></h4>
+                            <span class="font-weight-bold d-block mt-4">Hello,</span>
+                            <span>Your ouftit will be ready for pickup on your order date!</span>
+                            <div class="payment border-top mt-3 mb-3 border-bottom table-responsive">
+                                <table class="table table-borderless">
+                                <div class="product border-bottom table-responsive">
+                                    <table class="table table-borderless">
+                                    <tbody>
+                                        <tr>
+                                            <td width="20%">
+                                             <% @outfit.photos.each do |photo| %>
```

Comment:
The pictures on the side look great! Are we able to click on them to view them bigger? We mentioned having a photo carousel too! Not sure if you're able to implement that in a second version?